website/content/specification/syntax/instances.md

```diff
@@ -0,0 +1,466 @@
+---
+title: "Instances"
+Description: ""
+weight: 30
+---
+
+In a Metaschema module, complex information elements are created through composition. Through composition, an information element can be built by indicating which other information elements are used as its constituent parts.
+
+An *instance* is used to declare an information element *child* within a *parent* information element. In a Metaschema module, the parent information element is a definition, either a *field definition* or an *assembly definition*. The instance is a *flag instance*, *field instance*, or *assembly instance*, which in turn either references an existing [*top-level definition*](/specification/syntax/definitions/) by name or provides a [*inline definition*](/specification/syntax/inline-definitions/) as part of the instance declaration.
```

Comment:
Consider reversing this?

```suggestion
An *instance* is used to declare an information element *child* within a *parent* information element. Each instance is a *flag instance*, *field instance*, or *assembly instance*, and either references an existing [*top-level definition*](/specification/syntax/definitions/) by name or provides an [*inline definition*](/specification/syntax/inline-definitions/). In a Metaschema module, an instance appears inside the definition of its parent information element: an *assembly definition* may contain flag, field or assembly instances reflecting the objects to be permitted in that assembly, while a *field definition* may also include flag instances, defining its flags. Flag definitions, both *top-level* and *inline*, will never contain instances since flags have no children in the model, only values.
```

(Or have I only made it *more confusing*)?

Reply:
I'll apply the following, which slightly modifies your suggestion above.
```suggestion
An *instance* is used to declare an information element *child* within a *parent* information element. Each instance is a *flag instance*, *field instance*, or *assembly instance*, and either references an existing [*top-level definition*](/specification/syntax/definitions/) by name or provides an [*inline definition*](/specification/syntax/inline-definitions/).

In a Metaschema module, an instance appears inside the definition of its parent information element.
- An *assembly definition* may contain flag, field or assembly instances reflecting the objects to be permitted in that assembly.
- A *field definition* may only include flag instances.
- A Flag definition will never contain instances since flags have no children in the model, only values.
```